### DISCUSSION TITLE
Rename the `draw` function

### DISCUSSION THREAD

**Discussion by aleclarson** [radashi-org/radashi#93] [Ideas]
To me, “drawing” in this context implies the array will be mutated (like how a deck of cards depletes as you draw from it).

Possible new names:
1. `randomItem`
2. `randomElement`
3. `randomValue`

**Comment by Minhir** [comment]
What if we want to have similar functions for objects?  `randomKey` and `randomValue` fit object functions as well. 

I see two options:

1. Without polymorphism: `randomKey` and `randomValue` for objects, `randomElement` for arrays
2. With polymorphism: `randomKey` and `randomValue` for both arrays and objects
